src/nvim/ex_docmd.c

```diff
@@ -6861,7 +6905,26 @@ void ex_cd(exarg_T *eap)
     if (new_dir == NULL || vim_chdir(new_dir))
       EMSG(_(e_failed));
     else {
-      post_chdir(eap->cmdidx == CMD_lcd || eap->cmdidx == CMD_lchdir);
+      int scope = 0; // Depends on what command was invoked
```

Comment:
No, it could be through a bug somewhere else that `ex_cd` gets called by another ex command, let's say `:pwd` or something arbitrary, in which case the functions would still carry out as if we had called `:cd`. We could instead place an assertion at the top of the function
~~~~
assert(eap->cmdidx == CMD_cd || eap->cmdidx == CMD_chdir /* ... */
~~~~
Then we could assume the scope to be editor-wide by default.